config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
+  root 'users#index'
   get 'users/index'
   get 'users/show'
   get 'posts/index'
   get 'posts/show'
```

Comment:
### Optional
- [ ] Currently we have duplicated routes which is not the best way, also in line 3 - line 6, we can see the index and show page for users and posts, but we can also see that the routes are not nested as it should be. For example; a user must be present before a post can be shown, meaning we can see a post for a specific user. Let us kindly remove `line 3 - line 6` for a better UX